RocketApp/Domain/Sources/RocketsClient/RocketClient+liveValue.swift

```diff
@@ -10,13 +11,13 @@ extension RocketsClient: DependencyKey {
     return Self(
       getRocket: { id in
         Request(endpoint: URLs.baseURL + "/v3/rockets/\(id)").execute(using: networkClientType)
-          // TODO: Avoid ignoring the error by using ErrorReporting.
-          .ignoreFailure(setFailureType: RocketNetworkError.self)
+          .mapError({ RocketNetworkError(networkError: $0.cause) })
```

Comment:
Proc je tu to `({` to neni nutny ne? Je to proti konvencim...

Reply:
Smazu.

Reply:
Jeste jednou, pls, projdi si review, cely ho zpracuj (pripadne okomentuj cemu nerozumis) a pak reaguj. Tohle neni chat a me prijde notifikace ze je reakce na code review, ja tam jdu abych to znova prosel a tady je napsany "smazu" ale neni nic smazanyho.